src/js/views/globals/patient-modal/patient-modal_views.js

```diff
@@ -165,7 +165,7 @@ const PatientModal = View.extend({
     this.showLastNameView();
     this.showSexDroplist();
     this.showBirthDatePicker();
-    this.showWorkspacesComponent();
+    if (!this.model.isNew()) this.showWorkspacesComponent();
```

Comment:
I think this'd be better inside the function as a guard

Reply:
I agree, good call. I just updated it to that.